local/owr_video_renderer.h

```diff
@@ -62,6 +62,8 @@ GType owr_video_renderer_get_type(void) G_GNUC_CONST;
 
 OwrVideoRenderer *owr_video_renderer_new(const gchar *tag);
 
+void owr_video_renderer_set_request_context_callback(OwrVideoRenderer *renderer, GCallback callback, gpointer user_data);
```

Comment:
Instead of GCallback you probably want a proper-typed function pointer type here